code/modules/mob/living/simple_animal/abnormality/he/doomsday_calendar.dm

```diff
@@ -129,16 +135,17 @@
 
 /mob/living/simple_animal/hostile/abnormality/doomsday_calendar/PostWorkEffect(mob/living/carbon/human/user, work_type, pe)
 	if(work_type == ABNORMALITY_WORK_INSTINCT)
-		datum_reference.qliphoth_change(1)
+		datum_reference.qliphoth_change(4)//Work damage is multiplied by missing qliphoth counter, restore it fully.
 	bonusRed = 0 // Reset bonus red damage and the stat check.
 	return
 
 /mob/living/simple_animal/hostile/abnormality/doomsday_calendar/AttemptWork(mob/living/carbon/human/user, work_type)
 	if(work_type == ABNORMALITY_WORK_INSTINCT)
 		if(datum_reference.qliphoth_meter <= 4) // Sets bonus damage on instinct work.
-			bonusRed = (5 - (datum_reference.qliphoth_meter))//It samples your blood if it's below 2, less total damage than funpet
+			bonusRed = (5 - (datum_reference.qliphoth_meter))//It samples your blood if it's below the maximum counter, damage is RED instead of typeless
```

Comment:
This is never changed back to 0 if qliphoth_meter > 4. Also this conditional is pointless. It can totally just be `bonusRed = (5 - (datum_reference.qliphoth_meter))` because 5-5 is 0 and would be false.

Reply:
bonusRed is changed to 0 post-work regardless of qliphoth counter

Reply:
You are however correct about the condition. I'll go ahead and remove it